gradle-plugins/model-compiler/src/test/java/io/spine/gradle/compiler/rejection/given/Given.java

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright 2017, TeamDev Ltd. All rights reserved.
+ *
+ * Redistribution and use in source and/or binary forms, with or without
+ * modification, must retain the above copyright notice and the following
+ * disclaimer.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+ * A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+ * OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+ * SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+ * LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+ * DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+ * THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+package io.spine.gradle.compiler.rejection.given;
+
+import com.sun.javadoc.RootDoc;
+import io.spine.gradle.compiler.GradleProject;
+import org.junit.rules.TemporaryFolder;
+
+import java.util.Arrays;
+
+import static io.spine.gradle.compiler.Extension.getDefaultMainGenSpineDir;
+import static io.spine.gradle.compiler.util.JavaCode.toJavaFieldName;
+import static io.spine.gradle.compiler.util.JavaSources.getJavaExtension;
+
+/**
+ * @author Dmytro Grankin
+ */
+public class Given {
+
+    public static final String PROJECT_NAME = "rejections-gen-plugin-test";
+
+    /** Javadocs received from {@link RootDoc} contain "\n" line separator. */
+    @SuppressWarnings("HardcodedLineSeparator")
+    private static final String JAVADOC_LINE_SEPARATOR = "\n";
+    private static final String JAVA_PACKAGE = "io.spine.sample.rejections";
+    private static final String CLASS_COMMENT =
+            "The rejection definition to test Javadoc generation.";
+    private static final String REJECTION_NAME = "Rejection";
```

Comment:
Instead of exposing the constants as-is, it's cool to expose the methods, which produce something meaningful. Let's discuss.